packages/contracts/test/dollar/UbiquityChef.t.sol

```diff
@@ -6,11 +6,15 @@ import "../helpers/LiveTestHelper.sol";
 
 contract ZeroState is LiveTestHelper {
     event Deposit(
-        address indexed user, uint256 amount, uint256 indexed stakingShareId
+        address indexed user,
+        uint256 amount,
+        uint256 indexed stakingShareId
     );
 
     event Withdraw(
-        address indexed user, uint256 amount, uint256 indexed stakingShareId
+        address indexed user,
+        uint256 amount,
+        uint256 indexed stakingShareId
```

Comment:
What's up with the formatting?

Reply:
I have my Solidity formatter disabled, it's likely the output of 
Lint-Staged @pavlovcik

Reply:
This implies that we don't have our entire codebase formatted. We should run a quick formatting across the entire codebase then!

Reply:
> This implies that we don't have our entire codebase formatted. We should run a quick formatting across the entire codebase then!

that's right, our contracts aren't following the Prettier standard.